.github/ISSUE_TEMPLATE/rrfc.md

```diff
@@ -0,0 +1,34 @@
+---
+name: RRFC
+about: Requesting Request For Comment
```

Comment:
I would add more detail here about what this should be used for.
```suggestion
about: Requesting Request For Comment. If you have an idea for an RFC and want to get feedback before opening a formal RFC, use this issue template to describe your idea.
```
... or something.

And then how about adding a `config.yml` with a link to actually creating the RFC:
```
blank_issues_enabled: false
contact_links:
  - name: RFC
    url: https://github.com/lit/rfcs/edit/main/rfcs/0000-rfc-template.md
    about: To open a formal RFC, make a PR that copies the issue template linked here to a new file named with the format `NNNN-descriptive-name.md`. See [Opening an RFC](https://github.com/lit/rfcs#opening-an-rfc) for more details.
```

The /edit/ link above allows you to change the filename and make a PR, so seems like a nice in-browser link for making an RFC.

Reply:
I added the config.yml, but allowed blank issues for general discussion...